api/model.go

```diff
@@ -116,10 +116,19 @@ type EngineUpgradeInput struct {
 
 type Node struct {
 	client.Resource
-	Name            string          `json:"name"`
-	Address         string          `json:"address"`
-	AllowScheduling bool            `json:"allowScheduling"`
-	Status          types.NodeState `json:"status"`
+	Name            string              `json:"name"`
+	Address         string              `json:"address"`
+	AllowScheduling bool                `json:"allowScheduling"`
+	Disks           map[string]DiskInfo `json:"disks"`
+	Status          types.NodeState     `json:"status"`
```

Comment:
Just realized for that for the other objects, the status field is embedded. We should do the same with the node object.

Need a separate PR for that, and UI needs to be updated as well.

Filed https://github.com/rancher/longhorn/issues/103 for this.

Reply:
`types.NodeState` is a string object like `types.VolumeState`. I think we can't use it as an json object. Maybe just change the key from `Status` to `State` as Volume model does to keep consistency?

Reply:
It should be automatically converted to string. See `Volume.Frontend`.